lib/plaid/client.rb

```diff
@@ -14,33 +14,35 @@ class Client
 
     # Public: Set Plaid environment to use.
     #
-    # env - The Symbol (:tartan, :api), or a full String URL like
+    # env - The Symbol (:tartan, :production), or a full String URL like
     #       'https://tartan.plaid.com'.
     def env=(env)
       case env
       when :tartan, :api
```

Comment:
why do we have both `:api` and `:production`? It looks like they point to the same thing.